### DISCUSSION TITLE
What is the main Shad UI Documentation site built with?

### DISCUSSION THREAD

**Discussion by sambowenhughes** [shadcn-ui/ui#1878] [Q&A]
Does anybody know how the Shad UI Documentation site is built? Does it use Docusaurs?

**Comment by dewodt** [comment]
Based on Wapplyzer extension, it uses Nextjs.
![image](https://github.com/shadcn-ui/ui/assets/99950492/f7389d68-4fc8-4fd5-a316-596ce54ae9aa)
